build.gradle

```diff
@@ -1,7 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-	ext.kotlin_version = '1.1.3'
+	ext.
+			kotlin_version = '1.1.3'
```

Comment:
skoro update to powinien być od razu do najnowszej stabilnej wersji (chyba, że jest solidne uzasadnienie, że ma być inaczej), dla kotlina to `1.1.3-2`